src/main/java/edu/harvard/iq/dataverse/authorization/providers/oauth2/impl/OrcidOAuth2AP.java

```diff
@@ -78,7 +78,7 @@ public String getUserEndpoint( OAuth2AccessToken token )  {
     
     @Override
     public DefaultApi20 getApiInstance() {
-        return OrcidApi.instance( ! baseUserEndpoint.contains("sandbox") );
+        return OrcidApi.instance( isProduction() );
     }
     
     @Override
```

Comment:
This getUserRecord method is unchanged. Does that mean that the ORCID number rather than the full ORCID URL is still what we persist for the persistentuserid in the authenticateduserlookup table?

Update: I'm pretty sure we persist the ORCID like we always have, as a number. I see we have a getOrcidUrl method to change it to a URL.

(I know it's not a number. It has dashes and sometimes a z.)

Reply:
Yes - no change in that table (which also includes the authenticationproviderid column that makes it clear this is related to orcid). The new authenticatedorcid column in authenticateduser is using the URL form - which should allow using both the sandbox and production orcid over time/at the same time/ for these at least to not get mixed up (there's no second column to indicate sandbox/production in this table).

Reply:
Ok, thanks.